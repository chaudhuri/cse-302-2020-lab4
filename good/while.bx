// while loops

proc main() {
  var n = 20 : int;
  while (n > 1) {
    var i = 1 : int;
    var sum = 0 : int;
    while (i < n) {
      sum = sum + i;
      i = i + 1;
    }
    print(sum);
    print(n * (n - 1) / 2);
    n = n - 1;
  }
}