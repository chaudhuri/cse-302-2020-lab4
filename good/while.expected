190
190
171
171
153
153
136
136
120
120
105
105
91
91
78
78
66
66
55
55
45
45
36
36
28
28
21
21
15
15
10
10
6
6
3
3
1
1
